src/components/Card.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="card text-center q-ma-md q-pa-md shadow-up-3">
+    <slot class="card__image" name="image" />
+    <slot name="text" />
+    <hr />
+    <div class="card__data row inline content-start">
+      <slot class="card-data" name="data" />
+    </div>
+    <slot name="button" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Card'
+}
+</script>
+
+<style lang="scss">
+  .card {
+    max-width: 300px;
+    max-height: 74vh;
+
+    &__image {
+      width: 100%;
+      height: 25vh;
+    }
+
+     &__data {
+      color: #4a4a4a;
```

Comment:
essas cores são especificas ou vão serem utilizadas em outros lugares? pq se não for especifica pra o card, é bom utilizar sempre variáveis

Reply:
a classe __data se refere ao que exatamente? é importante o nome da classe fazer uma referencia sobre o local de utilização e confrontar se faz sentido chamar aquela parte de data..